types/yeoman-generator/index.d.ts

```diff
@@ -61,7 +61,9 @@ declare class Base extends EventEmitter {
 
     constructor(args: string|string[], options: {});
 
-    env: {};
+    env: {
```

Comment:
I think this type should be extracted as an `Environment` interface.

Reply:
Eh, don't think it's necessary if there's only one member right now.